src/main/java/net/fabricmc/loom/LoomRepositoryPlugin.java

```diff
@@ -69,11 +76,11 @@ public void apply(PluginAware target) {
 		});
 		repositories.maven(repo -> {
 			repo.setName("Fabric");
-			repo.setUrl("https://maven.fabricmc.net/");
+			repo.setUrl(MirrorUtil.getFabricRepository(finalAware));
```

Comment:
```suggestion
			repo.setUrl(MirrorUtil.getFabricRepository((ExtensionAware) target);
```
This should be simpler.

Reply:
Yes, thanks for the reminder, the new commit has completed the changes.